TinyMCE license key configuration docs (#2012)
plone/documentation: docs/classic-ui/tinymce-customization.md

the directory `profiles/default/registry` with arbitrary names ending in `.xml`.
This makes it easier to maintain and reuse your registry files.
```

> ## Use TinyMCE with a commercial license

Reviewer: ````suggestion
## Use TinyMCE with a commercial license

```{versionadded} Plone 6.2
```
````

> If you own a commercial TinyMCE license, you can provide the key in {menuselection}`Site Setup --> TinyMCE --> Advanced`

Reviewer: ```suggestion
If you own a commercial TinyMCE license, you can provide the key in {menuselection}`Site Setup --> TinyMCE --> Advanced`.
```

The TinyMCE 8 commercial license key will have a `T8LK:` prefix.

> See more information in the [TinyMCE licensing documentation](https://www.tiny.cloud/docs/tinymce/latest/license-key/)

Reviewer: ```suggestion
See more information in the [TinyMCE licensing documentation](https://www.tiny.cloud/docs/tinymce/latest/license-key/).
```

> If you have a TinyMCE license before version 8, read the important [License key migration documentation](https://www.tiny.cloud/docs/tinymce/latest/migration-from-7x/#license-key-system-update)

Reviewer: ```suggestion
If you have a TinyMCE license before version 8, read the important [License key migration documentation](https://www.tiny.cloud/docs/tinymce/latest/migration-from-7x/#license-key-system-update).
```

### Setup commercial license key manager plugin

> Since we cannot provide the license key manager plugin out of the box, because this is only shipped with a paid TinyMCE package, you have to provide the plugin in your own addon package.

Reviewer: ```suggestion
Since we can't provide the license key manager plugin out of the box, because this is only shipped with a paid TinyMCE package, you have to provide the plugin in your own add-on package.
```

See [Setting up the Commercial License Key Manager](https://www.tiny.cloud/docs/tinymce/latest/license-key/#setting-up-the-commercial-license-key-manager) for more information how to get the plugin.

> You have to provide the plugin file as a static resource and register it in {menuselection}`Site Setup --> TinyMCE --> Plugins and Toolbar -> Custom plugins` like this:

Reviewer: ```suggestion
You have to provide the plugin file as a static resource and register it in {menuselection}`Site Setup --> TinyMCE --> Plugins and Toolbar -> Custom plugins` as shown.
```

> ```

Reviewer: Default lexer when we don't have one.

````suggestion
```text
````

the directory `profiles/default/registry` with arbitrary names ending in `.xml`.
This makes it easier to maintain and reuse your registry files.
```

## Use TinyMCE with a commercial license

```{versionadded} Plone 6.2
```

> If you own a commercial TinyMCE license, you can provide the key in {menuselection}`Site Setup --> TinyMCE --> Advanced`.

Reviewer: ```suggestion
If you own a commercial TinyMCE license, you can enter the key in the control panel {menuselection}`Site Setup --> TinyMCE --> Advanced`.
```

The TinyMCE 8 commercial license key will have a `T8LK:` prefix.

See more information in the [TinyMCE licensing documentation](https://www.tiny.cloud/docs/tinymce/latest/license-key/).

If you have a TinyMCE license before version 8, read the important [License key migration documentation](https://www.tiny.cloud/docs/tinymce/latest/migration-from-7x/#license-key-system-update).

> ### Setup commercial license key manager plugin

Reviewer: ```suggestion
### Commercial license key manager plugin
```

## Use TinyMCE with a commercial license

```{versionadded} Plone 6.2
```

If you own a commercial TinyMCE license, you can provide the key in {menuselection}`Site Setup --> TinyMCE --> Advanced`.

The TinyMCE 8 commercial license key will have a `T8LK:` prefix.

See more information in the [TinyMCE licensing documentation](https://www.tiny.cloud/docs/tinymce/latest/license-key/).

If you have a TinyMCE license before version 8, read the important [License key migration documentation](https://www.tiny.cloud/docs/tinymce/latest/migration-from-7x/#license-key-system-update).

### Setup commercial license key manager plugin

Since we can't provide the license key manager plugin out of the box, because this is only shipped with a paid TinyMCE package, you have to provide the plugin in your own add-on package.

> See [Setting up the Commercial License Key Manager](https://www.tiny.cloud/docs/tinymce/latest/license-key/#setting-up-the-commercial-license-key-manager) for more information how to get the plugin.

Reviewer: ```suggestion
See [Setting up the Commercial License Key Manager](https://www.tiny.cloud/docs/tinymce/latest/license-key/#setting-up-the-commercial-license-key-manager) for more information of how to get the plugin.
```
